malasakit-django/feature_phone/admin.py

```diff
@@ -16,6 +23,45 @@ class RecordingAdmin(admin.ModelAdmin):
     }
 
 
+    def download_list_action(modeladmin, request, queryset):
+
+
+      def zip_file(zf, obj, file_field, folder=False):
```

Comment:
Is there any reason this function needs to be nested? If not, this should be moved into the module top-level.

Also, the linter should pick up on this, but the function needs a docstring, and indentation is four spaces.